build.gradle

```diff
@@ -1,89 +1,82 @@
-	compile('org.springframework.boot:spring-boot-starter-cache')
-	compile('org.springframework.retry:spring-retry')
-	compile('org.springframework.boot:spring-boot-starter-security')
-	compile('org.springframework.session:spring-session')
-	compile('org.springframework.boot:spring-boot-starter-web')
-	compile('org.springframework.boot:spring-boot-starter-websocket')
-	compile('joda-time:joda-time:2.9.9')
-	compile('io.javaslang:javaslang:2.0.4')
-    compile('com.ozwillo:ozwillo-java-spring-integration:1.26.0-dev')
-	compile('com.github.ben-manes.caffeine:caffeine:2.3.5')
-
-    testCompile('org.springframework.boot:spring-boot-starter-test')
-
+	implementation 'org.springframework.boot:spring-boot-starter-actuator'
+    implementation 'org.springframework.boot:spring-boot-starter-cache'
+    implementation 'org.springframework.boot:spring-boot-starter-data-mongodb'
+    implementation 'org.springframework.boot:spring-boot-starter-jersey'
+    implementation 'org.springframework.boot:spring-boot-starter-security'
+    implementation 'org.springframework.boot:spring-boot-starter-web'
+    implementation 'org.springframework.boot:spring-boot-starter-websocket'
+    implementation 'org.springframework.session:spring-session-core'
+    implementation 'io.javaslang:javaslang:2.0.4'
+    implementation 'com.github.ozwillo:ozwillo-java-spring-integration:1.26.0'
+    implementation 'com.github.ben-manes.caffeine:caffeine:2.3.5'
+    implementation 'com.squareup.retrofit2:retrofit:2.2.0'
+    implementation 'com.squareup.retrofit2:converter-jackson:2.2.0'
+    
+    testImplementation 'org.springframework.boot:spring-boot-starter-test'
+    testImplementation 'org.springframework.security:spring-security-test'
+	
     // required to parse logback-spring.groovy
     // TODO : quite a fat dependency for a configuration file
-    runtime('org.codehaus.groovy:groovy:2.4.7')
-
-	compile 'com.squareup.retrofit2:retrofit:2.2.0'
-	compile 'com.squareup.retrofit2:converter-jackson:2.2.0'
+    runtimeOnly 'org.codehaus.groovy:groovy:2.4.7'
+    runtimeOnly 'org.springframework.boot:spring-boot-devtools'
+}
 
+compileJava {
+    options.deprecation = true
 }
 
 bootRun {
-    // needed for hot reload of thymeleaf templates
-    addResources = true
-
-	environment SPRING_PROFILES_ACTIVE: environment.SPRING_PROFILES_ACTIVE ?: "dev"
+    environment SPRING_PROFILES_ACTIVE: environment.SPRING_PROFILES_ACTIVE ?: "dev"
 }
 
-springBoot {
-    // exclude Spring Boot Devtools from the fat jar
-    excludeDevtools = true
-
-    executable = true
-    embeddedLaunchScriptProperties =
-            [initInfoDescription: 'Ozwillo Datacore Exporter',
-             initInfoShortDescription: 'Ozwillo Datacore Exporter',
-             initInfoProvides: 'ozwillo-dc-exporter']
+bootJar {
+    launchScript({
+        properties 'initInfoProvides': 'ozwillo-dc-exporter'
+    })
 }
 
-task wrapper(type: Wrapper) {
-	gradleVersion = '2.14'
+wrapper {
+	gradleVersion = '4.9'
```

Comment:
You're right. This task has been removed

Gradle-4.9 because version upgrade started on Aug. 24 while gradle-4.10 was released on Aug. 27
Gradle is now on version 4.10